components/support/jwcrypto/src/aes.rs

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Wrappers we use around rc_crypto's AES implementation.
+
+use crate::{
+    error::{JwCryptoError, Result},
+    CompactJwe, EncryptionAlgorithm, JweHeader,
+};
+use rc_crypto::{aead, rand};
+
+/// Does the AES-encrypt heavy-lifting for the schemes supported by this crate.
```

Comment:
Specifically, you are implementing "enc=A256GCM" from RFC7518, Section 4.7. I think it's worth saying that explicitly in the comments somewhere here, and perhaps even linking out to it [here](https://tools.ietf.org/html/rfc7518#section-4.7) for ease of reference.